src/mrpro/data/AcqInfo.py

```diff
@@ -140,26 +148,34 @@ class AcqInfo(MoveDataMixin):
     scan_counter: torch.Tensor
```

Comment:
Currently not. I would say we remove it and reimplement it in case we realize we do need it.

Reply:
@ckolbPTB did you resolve the conversation because you want to keep it?
i don#t mind either way. pretty sure I will not cause any issues to keep it, but I don't see a use for it